package/MDAnalysis/lib/util.py

```diff
@@ -2560,3 +2561,14 @@ def no_copy_shim():
     else:
         copy = False
     return copy
+  
+  
+def is_installed(modulename: str):
```

Comment:
If this a new function, add an explicit test for it.

I think, at the moment we get coverage for it because it is used in code but that's not a test of the function itself.

Reply:
Added a test.